server/app.js

```diff
@@ -37,3 +53,29 @@ app.get("/proxy/:proxyUrl*", (req, res) => {
 app.listen(port, () => {
   console.log("Server started on port 3000!");
 });
+
+app.get("/repos", async (_, res) => {
+  const response = await octokit.paginate("GET /orgs/uclaacm/repos", {
+    per_page: 100,
+  });
+
+  const result = new Map();
+  result.set("no-topic", []);
+
+  for (let obj of response) {
+    if (!!obj.homepage) {
+      if (obj.topics.length) {
+        for (let topic of obj.topics) {
+          if (result.has(topic)) {
+            console.log(result.get(topic))
+            result.set(topic, [...result.get(topic), obj.homepage]);
+          }
+          else result.set(topic, [obj.homepage]);
+        }
+      } else {
+        result.set("no-topic", [...result.get("no-topic"), obj.homepage]);
+      }
+    }
+  }
+  return res.json(Object.fromEntries(result));
```

Comment:
Inside of endpoints that we hit, it's common practice that instead of doing a return statement explicitly for our request handler, that we send the status of our request with the given data like: `res.status(200).send(OUR_DATA)` instead of the `return OUR_DATA;` since it makes it easier for people reading our code to know the return codes of our endpoints, especially for if we do error handling in the future!